### PR TITLE
Changed Swift stubs file naming to avoid file naming conflicts

### PR DESCRIPTION
**Pull request by mjjimenez**
I'm getting file naming conflicts from the 2 placeholder protobuf files when turned into swift files. Xcode doesn't differentiate between files in different folders. I've changed the file naming option generation for swift stubs to use the file path with underscores as the file name.

So `foo/bar/baz.proto` becomes `foo_bar_baz.pb.swift` and no directories are generated.

### CODE REVIEW

File: .github/workflows/swift-stubs.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Build Swift Stubs
```
Comment (karlhigley) [inline .github/workflows/swift-stubs.yml:1]: This got combined into [a unified workflow](https://github.com/OpenMined/syft-proto/blob/master/.github/workflows/stubs-and-publish.yml) that builds the stubs for all the supported languages. 😃

**Review by karlhigley — COMMENTED** [review]
What are the two `Placeholder` proto files? There should only be one.

File: .github/workflows/swift-stubs.yml (hunk 1 above)
Comment (mjjimenez) [reply .github/workflows/swift-stubs.yml:1]: I think I messed-up pulling from upstream in my fork. I'll probably close this one and open another pull-request.

**Comment by mjjimenez** [comment]
Hmm, you're right @karlhigley . There's 2 swift stub files in `frameworks/torch/tensors/interpreters/v1/` and another one in `execution/v1`. But if I check the `proto` files, there's only one `placeholder.proto` in `execution/v1`. I guess there was something moved / deleted / renamed in interpreters?  

I'll just close this pull-request and delete duplicate `placeholder.swift` in another pull-request after I fix my fork. Maybe I should just delete the contents of the `swift` folder before generating the stubs so that this shouldn't happen again.

**Comment by karlhigley** [comment]
@mjjimenez We do need a solution for this, because it's happened with the Python stubs too. Possible that the stub generating workflows should all delete the stubs before regenerating?

**Pull request closed without merging**